agent/06_agent_create_cluster.sh

```diff
@@ -528,17 +591,24 @@ if [[ ! -z $AGENT_OPERATORS ]]; then
     put_operator_file
 fi
 
-wait_for_cluster_ready
+# Current goal is to only verify if the nodes are booted fine,
+# TUI sets the rendezvous IP correctly and UI is accessible.
+# The next goal is to simulate adding the cluster details via UI
+# and complete the cluster installation.
+if [[ "${AGENT_E2E_TEST_BOOT_MODE}" != "ISO_NO_REGISTRY" ]]; then
+    wait_for_cluster_ready
```

Comment:
maybe an exit 0 would be sufficient to temporary skip those parts not yet applicable for the `ISO_NO_REGISTRY`